### PR TITLE
Fix some spelling mistakes

### PR DESCRIPTION
**Pull request by plewin**

### CODE REVIEW

File: presentation/index.html [hunk 1]
```diff
@@ -261,7 +261,7 @@
 * Manage users / contacts access credentials for the backend / WebUI <!-- .element: class="fragment" data-fragment-index="2" -->
 
 Realms allow to create *parts* in the monitoring configuration that will be considered separately by Alignak. <!-- .element: class="fragment" data-fragment-index="3" -->
-As an example, a whole company infrastructure can be splitted in two realms: France and Italy. <!-- .element: class="fragment" data-fragment-index="4" -->
+As an example, a whole company infrastructure can be spit in two realms: France and Italy. <!-- .element: class="fragment" data-fragment-index="4" -->
```
Comment (mohierf) [inline presentation/index.html]: spit, hmm... split would be better not 😆

Comment (plewin) [inline presentation/index.html]: oops 😅

**Comment by mohierf** [comment]
@ddurieux : would you mind merging this PR?

**Comment by mohierf** [comment]
@ddurieux ???

**Pull request merged**